scripts/setup-ubuntu.sh

```diff
@@ -49,9 +52,11 @@ sudo --preserve-env apt update && apt install -y \
   libzstd-dev \
   libre2-dev \
   libsnappy-dev \
+  libthrift-dev \
```

Comment:
Would this affect docker images generated using this script ?

Reply:
Yes they will also have thrift installed, which should save build time when parquet is active (as arrow needs thrift).